### ISSUE TITLE
Album Cover - cannot choose or use CoverArt Archive

### ISSUE THREAD

**Issue by Gwend4l** [CDrummond/cantata#744]
Hi!

It seems that I can't make Cantata find covers for many albums that only have covers on CoverArt Archive (https://archive.org/details/coverartarchive).

For instance:
![capture du 2015-10-07 11-32-22](https://cloud.githubusercontent.com/assets/1388961/10333907/20da5678-6ce7-11e5-9e19-3f2d53ba1912.png)

On the right, I have Cantata opened. I chose "Set Image" for the album, and the window to fetch a cover opens and doesn't find anything. I try to change the provider, but nothing happens when I pick "CoverArt Archive" or any other: no search, nothing displayed neither in the window nor in the list.

On the left, I have CoverArt Archive opened in which I did the exact same string search, and the cover can be found.

Maybe I did something wrong?

**Comment by CDrummond** [comment]
CoverArt archive needs last.fm as well. Searching on cover-art uses the musicbrainz id - and Cantata gets this from last.fm

That being said, the last.fm code was broken - or more likely the last.fm response has changed. This is fixed in git/master (for 2.0) now. There will be no more 1.x releases.

**Comment by Gwend4l** [comment]
Thanks for the fast answer and for the fix :)

Just being curious, why don't you directly use the API provided by musicbrainz to search for the id of an album? (example here https://stackoverflow.com/questions/13453678/how-to-get-album-image-using-musicbrainz/13459240#13459240)

For instance in the case I've shown in my screenshot, the following query seems to work:
~~~
https://www.musicbrainz.org/ws/2/release/?query=artist:DDRKirby\%28ISQ\%29%20AND%20release:The%20Ecstasy%20of%20Life
~~~

also why not using libmusicbrainz (again, just being curious)? https://metabrainz.github.io/libmusicbrainz/5.1.0/class_music_brainz5_1_1_c_query.html#abe151c7483ac026df73bd76c23d9b9dd
